easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'pandas'
+version = '1.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://pypi.python.org/pypi/pandas/"
+description = """pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures
+ and data analysis tools for the Python programming language."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('numexpr', '2.7.1', {
```

Comment:
We have specific easyconfigs for `numexpr` and version 2.7.1 is the one used in `2020a`. So I would rather add `numexpr-2.7.1-foss-2020a-Python-3.8.2.eb` and depend on it, than bundle `numexpr` here.

Reply:
done in 1fc0562